nx584/config.yaml

```diff
@@ -11,3 +14,37 @@ arch:
   - i386
 ports:
   5007/tcp: 5007
+options:
+  serial:
+   enabled: true
+   device: "/dev/serial/by-id/usb-Prolific_Technology_Inc._USB-Serial_Controller-if00-port0"
+   baud: "9600"
+  socat:
+   enabled: false
+   host: "192.168.30.1"
```

Comment:
Also here - let's change it to "192.168.0.1" which is kinda like the most popular network?